src/app/components/layouts/tag/terra-tag.component.ts

```diff
@@ -22,10 +26,37 @@ export class TerraTagComponent
     @Input()
     public inputCustomClass:string;
 
-    constructor()
+    @Input()
+    public inputOwnColor:string;
+
+    constructor(private _element:ElementRef)
     {
         this.inputIsTagged = false;
         this.inputIsTaggable = false;
+        this.inputOwnColor = null;
         this.inputCustomClass = null;
     }
+
+    public ngAfterViewInit():void
```

Comment:
wenn es ein festes Set an Farben gibt, kann die Zurodnung der text Farbe ebenfalls in scss erfolgen